stan/math/rev/scal/fun/is_nan.hpp

```diff
@@ -17,7 +17,10 @@ namespace math {
  *
  * @return <code>1</code> if the value is NaN and <code>0</code> otherwise.
  */
-inline bool is_nan(const var& v) { return is_nan(v.val()); }
+template <typename T, require_var_t<T>...>
+inline bool is_nan(T&& v) {
+  return is_nan(std::forward<decltype(v.val())>(v.val()));
```

Comment:
Can this be simplified to ``` std::forward<double>``` since we know the type of the ```var``` value?

Reply:
Yes!